### PR TITLE
fix(webgpu): Fix `GPUAdapter#isFallbackAdapter` and `GPUAdapter#info` properties

### PR DESCRIPTION
**Pull request by petamoriken**
Fixed `GPUAdapter` bugs:

* `GPUAdapter#isFallbackAdapter` being `undefined`
* `GPUAdapter#info` throwing `TypeError`
  * introduced by #24783
* `GPUAdapter#info` closing adapter resources
  * introduced by #23752

### CODE REVIEW

**Comment by petamoriken** [comment]
@littledivy @crowlKats Please take a look!

**Review by littledivy — APPROVED** [review]
LGTM

File: ext/webgpu/01_webgpu.js [hunk 1]
```diff
@@ -417,14 +417,15 @@ function createGPUAdapter(inner) {
...
 class GPUAdapter {
   /** @type {InnerGPUAdapter} */
   [_adapter];
-  /** @type {bool} */
-  [_invalid];
   /** @type {GPUAdapterInfo | undefined} */
-  #adapterInfo;
+  [_adapterInfo];
+  /** @type {boolean} */
+  [_invalid] = false;
```
Comment (crowlKats) [inline ext/webgpu/01_webgpu.js:428]: This default value will not work

Comment (petamoriken) [reply ext/webgpu/01_webgpu.js]: Indeed... Fixed 👍

**Review by crowlKats — APPROVED** [review]
LGTM

**Pull request merged**